Hugo Cullinan/week01/day-3/js-homework-1.js

```diff
@@ -0,0 +1,84 @@
+// 1. Define a function `maxOfTwoNumbers` that takes two numbers as arguments and returns the largest of them. Use the if-then-else construct available in Javascript. You'll have to remember your pre-work, or do some googling to figure this out.
+const maxOfTwoNumbers = function (a, b) {
+  if (a > b) {
+    return a;
+  }else {
+    return b;
+  }
+}
+// 2. Define a function `maxOfThree` that takes three numbers as arguments and returns the largest of them.
+const maxOfThree = function (a, b, c) {
+  if (a > b && a > c) {
+    return a;
+  }else if (b > c && b > a) {
+    return b;
+  }else {
+    return c;
+  }
+}
+// 3. Write a function that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise.
+const isVowel = function (char) {
+  const vowels = 'aeiou';
+  if (char.length === 1) {
+    if (vowels.includes(char)) {
+      return true;
+    }else {
+      return false;
+    }
+  }else {
+    return "Please give a one character string!"
+  }
+}
```

Comment:
This is awesome!! Love how you went about this. 
Also love the addition if/else statement to make sure only one character is entered.